src/Packages/Passport/Runtime/ThirdParty/Gree/Assets/Plugins/unity-webview-webgl-plugin.jslib

```diff
@@ -0,0 +1,36 @@
+mergeInto(LibraryManager.library, {
+	_gree_unity_webview_init: function(name) {
+		var stringify = (UTF8ToString === undefined) ? Pointer_stringify : UTF8ToString;
+		unityWebView.init(stringify(name));
+	},
+
+	_gree_unity_webview_setMargins: function (name, left, top, right, bottom) {
```

Comment:
Can we remove _gree_unity_webview_setMargins and _gree_unity_webview_setVisibility?